src/web/extractors/rustdoc.rs

```diff
@@ -236,6 +241,16 @@ impl RustdocParams {
         })
     }
 
+    pub(crate) fn confirmed_name(&self) -> Option<&KrateName> {
+        self.confirmed_name.as_ref()
+    }
+    pub(crate) fn with_confirmed_name(self, confirmed_name: Option<impl Into<KrateName>>) -> Self {
```

Comment:
Why implementing `Into`/`From` instead of `TryInto` if the `ConfirmedName` conversion can fail?

Reply:
thinking about it, probably to make using the builder easier, not having to handle errors. 

In the cases where we use the builder, we already have a parsed object, and the `Into<` arg is mostly there so automatically clone if you pass a ref, and another `From<` impl I forgot :D

Reply:
the current callers, yes. 

I think I'm probably more used to builders always accepting `Into<T>` as a default, for general ease of use. 

I'm happy to go either way (no trait, or tryinto), whatever you prefer :)